libs/indibase/inditelescope.cpp

```diff
@@ -410,8 +410,9 @@ bool Telescope::updateProperties()
         if (HasPierSideSimulation())
         {
             defineSwitch(&SimulatePierSideSP);
-            // ensure that all properties are set
-            setSimulatePierSide(m_simulatePierSide);
+            loadConfig(true, "SIMULATE_PIER_SIDE");
+            int index = IUFindOnSwitchIndex(&SimulatePierSideSP);
```

Comment:
Perhaps a better method is to use IUGetConfigSwitch to immediately read back the switch value from the config file? loadConfig does not immediately trigger update of the property in question.